app/lib/builder/components/CodeEditor/extensions/autocomplete.js

```diff
@@ -13,103 +13,19 @@ const Completion_Label = (value) => {
   }
 }
 
-function svelteCompletions(data) { 
-  const completions = [
-    snippetCompletion('{#if ${true}}\n\t${<span>Shown if true</span>}\n{:else}\n\t${<span>Shown if false</span>}\n{/if', {
-      label: "{#if}",
-      type: "text",
-      detail: "Conditionally render a block of content",
-    }),
-    snippetCompletion('{#each ${["one", "two"]} as ${item}}\n\t${\{item\\}}\n{/each', {
-      label: "{#each}", 
-      type: "text",
-      detail: "Loop over array or Repeater items"
-    }),
-    snippetCompletion('{#await ${promise}}\n\t${<span>promise is pending</span>}\n{:then ${value}}\n\t${<span>promise was fullfilled</span>}\n{:catch ${error}}\n\t${<span>promise was rejected</span>}\n{/await', {
-      label: "{#await}", 
-      type: "text",
-      detail: "Show content depending on the states of a Promise"
-    }),
-    snippetCompletion('{#key ${"value"}}\n\t<span>this will re-render when "value" changes</span>\n{/key', {
-      label: "{#key}", 
-      type: "text", 
-      detail: "Re-render a block when a value changes"
-    }),
-    snippetCompletion('{@html ${"<p>content</p>"}', {
-      label: "{@html}", type: "text", detail: "Render HTML from a Markdown field"
-    }),
-    snippetCompletion('{@debug ${variable}', {
-      label: "{@debug}", 
-      type: "text", 
-      detail: "Log a variable's value"
-    }),
-    snippetCompletion('{@const ${variable = "foo"}', {
-      label: "{@const}", 
-      type: "text", 
-      detail: "Define a local constant"
-    }),
-  ]
-  return svelteLanguage.data.of({
-    autocomplete: (context) => {
-      const word = context.matchBefore(/\S*/)
-
-      // Svelte blocks
-      if ((word.text.includes('{#') || word.text.includes('{@'))) {
-        const position = (word.text.indexOf('{#') !== - 1 ? word.text.indexOf('{#') : word.text.indexOf('{@')) 
-        return {
-          from: word.from + position,
-          options: completions
-        }
-      }
+function svelteCompletions(completions) {
+  // Return an autocompletion source function
+  return (context) => {
+    const word = context.matchBefore(/\{[^}]*/)
```

Comment:
The regex could be changed like below to allow autocompleting field key (eg. heading) in emmet shorthands like `div{{heading}}` (which is completed into `<div>{heading}</div>`).

```suggestion
    const word = context.matchBefore(/\{[a-zA-Z0-9_]*/)
```